src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -27,6 +30,10 @@ public class AddressBook implements ReadOnlyAddressBook {
         persons = new UniquePersonList();
     }
 
+    {
+        tags = new UniqueTagList();
```

Comment:
Is there a need for this initialization block to be separate from the one at line 30? Can they be combined?

Reply:
Made the changes. Thanks